tutorials/admin/troubleshooting/faq.md

```diff
@@ -0,0 +1,18 @@
+# Frequently Asked Questions
+
+## I cannot run docker as a non-root user
```

Comment:
I'd try H3 or H4. H2 looks too big.

Reply:
I've experimented with this and jupyterbook simply won't allow us to do so because of:

> Headers should increase linearly. If you’re inside of a section with one `#`, then the next nested section should start with `##`. Avoid jumping straight from `#` to `###`

https://jupyterbook.org/en/stable/file-types/index.html#rules-for-all-content-types

There are ways of doing it with embedding html directly but then the table of contents (https://github.com/DACCS-Climate/marble-tutorials/pull/29#discussion_r1467046078) doesn't render correctly. I'd rather have a table of contents and larger headers than smaller headers and no table of contents.